README.md

```diff
@@ -34,6 +34,22 @@ where possible options include:
     -v, -version                    print version
 ```
 
+# How to use from Gradle
+
+Dukat will automatically run for npm dependencies by specifying the following in `build.gradle`:
+```
+kotlin.js.experimental.generateKotlinExternals=true
+```
+
+The version of dukat can be specified in build.gradle.kts by adding the following:
+```
+import org.jetbrains.kotlin.gradle.targets.js.nodejs.NodeJsRootPlugin
+...
+NodeJsRootPlugin.apply(project).apply {
```

Comment:
Instead of applying the root plugin manually, you should just configure the extension like
```kt
configure<NodeJsRootExtension> {
    versions.dukat.version = "0.5.7"
}
```
Which should probably also be shown in the Kotlin/JS Gradle plugin documentation.